hvantk/ptm/plot.py

```
@@ -84,34 +84,69 @@ def plot_landscape_summary(
+    ax1.set_title("Variant Counts")
+    ax1.legend(frameon=False)
+    ci_hi_str = (
+        f"{result.enrichment_ci_high:.2f}"
+        if result.enrichment_ci_high < 1e6 else "∞"
+    )
+    ax1.text(
         0.98, 0.95,
-        f"Fisher OR={result.enrichment_odds_ratio:.2f}, p={result.enrichment_p_value:.2e}",
-        transform=ax.transAxes, ha="right", va="top", fontsize=9,
+        f"OR={result.enrichment_odds_ratio:.2f} "
+        f"({result.enrichment_ci_low:.2f}–{ci_hi_str})\n"
```

Ruff (0.15.7)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)